src/test/java/org/socialsignin/spring/data/dynamodb/config/DynamoDBAuditingBeanDefinitionParserTest.java

```diff
@@ -19,6 +19,7 @@
 import javax.xml.parsers.DocumentBuilderFactory;
 
 import org.junit.Before;
+import org.junit.Ignore;
```

Comment:
You probably didn't intend for this to come in

Reply:
Agreed